{
  "Header": {
    "Original": "CVE-2022-2873 (An out-of-bounds memory access flaw was found in the Linux kernel Inte ...)",
    "Line": 62023,
    "ID": "CVE-2022-2873",
    "Description": "(An out-of-bounds memory access flaw was found in the Linux kernel Inte ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-5324-1 DLA-3403-1 DLA-3349-1}",
      "Line": 62024,
      "Type": "xref",
      "Bugs": [
        "DSA-5324-1",
        "DLA-3403-1",
        "DLA-3349-1"
      ]
    },
    {
      "Original": "- linux 6.1.4-1",
      "Line": 62025,
      "Type": "package",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.1.4-1"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=2119048",
      "Line": 62026,
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=2119048"
    },
    {
      "Original": "NOTE: https://lore.kernel.org/lkml/20220729093451.551672-1-zheyuma97@gmail.com/T/",
      "Line": 62027,
      "Type": "NOTE",
      "Description": "https://lore.kernel.org/lkml/20220729093451.551672-1-zheyuma97@gmail.com/T/"
    },
    {
      "Original": "NOTE: https://git.kernel.org/linus/39244cc754829bf707dccd12e2ce37510f5b1f8d",
      "Line": 62028,
      "Type": "NOTE",
      "Description": "https://git.kernel.org/linus/39244cc754829bf707dccd12e2ce37510f5b1f8d"
    }
  ]
}